lib/chef/resource/habitat_install.rb

```diff
@@ -25,6 +25,18 @@ class HabitatInstall < Chef::Resource
       unified_mode true
       provides :habitat_install
 
+      description "This resource will install the newest stable version of habitat"
+
+      property :name, String, default: ""
```

Comment:
Make sure to set description fields on all of the properties. We use that to generate docs.

Reply:
was working on that this morning. seems to be causing me some issues so i'm going back through them now